### DISCUSSION TITLE
How to set scss file to use 2 space indent size

### DISCUSSION THREAD

**Discussion by kzhui125** [zed-industries/zed#18400] [Feature Requests]
How to set scss file to use 2 space indent size
Thanks,

**Comment by omarjatoi** [comment]
You can use the tab_size setting in configuration: https://zed.dev/docs/configuring-zed#tab-size

```
"languages": {
  "SCSS": {
    "tab_size": 2
  }
}
```

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
